en/docs/administer/product-configurations/adding-internationalization.md

```diff
@@ -1,214 +1,153 @@
 # Adding Internationalization and Localization
 
-The API Manager includes two Web interfaces as API Publisher and API Store. The steps below explain how to localize the API Publisher UI. Similarly, the API Store could also be localized by following the steps below.
+The API Manager includes two Web interfaces, namely the API Publisher and Developer Portal. The steps below explain how you can localize the API Publisher and the Developer Portal.
+
+## Changing the browser language
```

Comment:
3.0 default devportal shipps no additional language other than English. 
the publisher has English, Spanish, and French

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1093/commits/f2775c2ecf29dda5bd8f2912cae4c53a840597cb